src/rebar_paths.erl

```diff
@@ -0,0 +1,210 @@
+%% BEFORE THIS FIX: rebar3 ct  266.78s user 144.06s system 144% cpu 4:33.70 total
+%% CURRENT TIME:    rebar3 ct  419.30s user 301.00s system 152% cpu 7:51.98 total
```

Comment:
Are these still accurate and do we want to keep them in the comment like this?

Reply:
Nah I'll take them out. End results are on par with what it was before; I forgot to take it out after resolving performance issues.